wave_utils/src/math.cpp

```diff
@@ -86,4 +86,12 @@ double wrapTo360(double euler_angle) {
     }
 }
 
+Mat3 skew_symmetric_matrix(Vec3 a){
+  Mat3 skew_matrix;
+  skew_matrix << 0, -a(2), a(1),
```

Comment:
If you're using clang-format it will put the following in 1 line, consider doing this:

```
  Mat3 skew_matrix;
  // clang-format off
  skew_matrix << 0, -a(2), a(1),
                a(2), 0, -a(0),
                -a(1), a(0), 0;
  // clang-format on
```